src/Components/ToDo/index.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+
+import Typography from '@material-ui/core/Typography';
+import Container from '@material-ui/core/Container';
+
+import store from '../../Components/ToDo/SagaStore/store';
+
+import AddToDo from './AddToDo';
+import ToDoList from './ToDoList';
+// import ResetButton from './resetButton';
+
+const ToDo = () => (
+  <Provider store={store}>
```

Comment:
use store Provider in src/index.tsx file

Reply:
Sure, I'll update it to entry!